### DISCUSSION TITLE
Remove camera from Frigate+

### DISCUSSION THREAD

**Discussion by greghesp** [blakeblackshear/frigate#9627] [Ask A Question]
I setup some test cameras to test some functionality a while ago, and now I have a `test_cam` in Frigate+ that I cant get rid of.

Any ideas how I can do this?

**Comment by blakeblackshear** [comment]
There isn't a delete function yet. Having them there doesn't hurt anything.

**Reply by nibagit** [reply]
Rename option be great as well.  Thanks on a great product, just got my Frigate+ subscription and wanted to cleanup my old cameras from prior config!!

**Reply by zoechi** [reply]
It's quite cumbersome, because I renamed mine sightly and now they are hard to distinguish.
